nvtabular/framework_utils/tensorflow/layers/embedding.py

```diff
@@ -198,7 +185,7 @@ def __init__(self, feature_columns, aggregation="concat", name=None, **kwargs):
         super(DenseFeatures, self).__init__(name=name, **kwargs)
 
     def build(self, input_shapes):
-        assert all(shape[1] == 1 for shape in input_shapes.values())
+        # assert all(shape[1] == 1 for shape in input_shapes.values())
```

Comment:
If we don't need these asserts anymore, we should just delete rather than comment out
```suggestion
```

Reply:
We might still want to assert some version of this for the tensors inside the tuple?